shielder/contract/traits/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod psp22;
```

Comment:
why do we copy the whole PSP22 API instead of reusing psp22 crate? if it's the ink version conflict, I would rather move this code there - keeping it here is not a good idea (we pollute shielder code and might diverge from any development that is done in the reference repo)